src/interactive/AccordionPanel.jsx

```diff
@@ -23,6 +23,7 @@ class AccordionPanel extends React.Component {
 
 		this.state = {
 			height: props.isOpen ? 'auto' : '0px'
+			// hasFocus: false
```

Comment:
let's kill this state property instead of commenting it out

Reply:
lolwoops